From 47181f9831bdb849188cac349e47f039d2bc91b8 Mon Sep 17 00:00:00 2001
From: Mathieu Bridon <bochecha@daitauha.fr>
Date: Tue, 16 May 2017 22:16:17 +0200
Subject: [PATCH 2/4] Move the 128x128 icon

${prefix}/share/pixmaps is a legacy path for icons and probably
shouldn't be used any more at this point.

Since this is a 128x128 icon, it deserves its place in
${prefix}/share/hicolor/128x128/apps/, so that desktops like GNOME Shell
can display a bigger icon than the 32x32 one which would be used
otherwise.
---
 .../hicolor/128x128/apps}/playitslowly.png                  | Bin
 1 file changed, 0 insertions(+), 0 deletions(-)
 rename share/{pixmaps => icons/hicolor/128x128/apps}/playitslowly.png (100%)

diff --git a/share/pixmaps/playitslowly.png b/share/icons/hicolor/128x128/apps/playitslowly.png
similarity index 100%
rename from share/pixmaps/playitslowly.png
rename to share/icons/hicolor/128x128/apps/playitslowly.png
-- 
2.9.4

